docs/enhancements/README.md

```diff
@@ -0,0 +1,31 @@
+# Enhancements Tracking for Assisted Installer
```

Comment:
I suggest adding a section addressing when to create an OpenShift enhancement proposal instead. Here are some indicators that could be included in a "rough heuristic":
* requires changes in OpenShift
* requires changes in and/or consensus with other components related to OpenShift
* substantially impacts the requirements for and/or experience of installing and provisioning OpenShift
* would benefit from approval by OpenShift architects